### PR TITLE
[feat]: Adding SignUp Page For Ecom

### PR DESCRIPTION
**Pull request by raghav2110**
Hi, I have added signup page for ecom. Please review my PR.

### CODE REVIEW

File: css/signupPage.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+.email-label{
+    display: block;
+    font-size: 1rem;
+    color: var(--headerbg);
+    font-weight: bold;
+    margin: 0.5rem 0;
+}
+
+input[type=text]input[type=password].name-input{
+    border: 0.2rem solid #f3f4f6;
```
Comment (ZuberDunge) [inline css/signupPage.css]: use color variables if possible

**Review by ZuberDunge — APPROVED** [review]
Everything looks perfectly fine. Can merge. Also, UI looks good.

**Review by Hamdrive — APPROVED** [review]
Code looks good and clean
Can be merged!

File: css/signupPage.css (hunk 1 above)
Comment (raghav2110) [reply css/signupPage.css]: Resolved that change

**Pull request merged**